sunpy/visualization/animator/wcs.py

```diff
@@ -79,7 +78,7 @@ def __init__(self, data, wcs, slices, coord_params=None, ylim='dynamic', ylabel=
 
         if self.plot_dimensionality == 1:
             try:
-                from astropy.visualization.wcsaxes.frame import RectangularFrame1D
+                pass
```

Comment:
```suggestion
                from astropy.visualization.wcsaxes.frame import RectangularFrame1D # NOQA
```

Reply:
why on earth does autoflake8 do this?! It's fucking mad.

Reply:
Is the import not used?

Reply:
An import in a `try ... except ImportError` block clearly is used. why would you ever replace that with `pass` :scream: